lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -49,34 +49,40 @@ class StartUrl extends Gatherer {
   pass(options) {
     return options.driver.getAppManifest()
       .then(response => {
-        return manifestParser(response.data, response.url, options.url);
+        return response && manifestParser(response.data, response.url, options.url);
       })
       .then(manifest => {
-        if (!manifest.value.start_url || !manifest.value.start_url.raw) {
-          return Promise.reject(new Error(`No web app manifest found on page ${options.url}`));
+        if (!manifest || !manifest.value) {
+          this.debugString = `No usable web app manifest found on page ${options.url}`;
```

Comment:
if there's a `manifest` but no `manifest.value`, there will be a `manifest.debugString` probably worth keeping